### PR TITLE
coop stats

### PR DESCRIPTION
**Pull request by rfomin**
I implement PrBoom+'s killcount method because without it, the Time/STS widget looks broken in co-op. Also we have this code for demo compatibility anyway.

### CODE REVIEW

File: Source/hu_stuff.c [hunk 1]
```diff
@@ -805,25 +805,67 @@ static int HU_top(int i, int idx1, int top1)
...
+    if (playeringame[i])
+    {
+      if (playerscount == 0)
+      {
+        offset = sprintf(kills_str,
+          "\x1b%c%d", color, players[i].killcount);
+      }
+      else
+      {
+        offset += sprintf(kills_str + offset,
```
Comment (fabiangreffrath) [inline Source/hu_stuff.c:829]: Isn't `offset` definitely still `0` here?

File: Source/p_inter.c [hunk 2]
```diff
@@ -695,26 +695,37 @@ static void P_KillMobj(mobj_t *source, mobj_t *target)
...
+            for (i = 0, playerscount = 0; i < MAXPLAYERS; ++i)
             {
               if (playeringame[i])
               {
-                P_Random(pr_friends);
-                break;
+                if (demo_version < 203)
+                {
+                  players[i].killcount++;
+                  break;
```
Comment (rfomin) [inline Source/p_inter.c:716]: PrBoom+ gives all kills to the first player in vanilla demos. Maybe we can randomize it here too, but with `Woof_Random` function?

Comment (fabiangreffrath) [reply Source/p_inter.c:716]: Yes, okay.

File: Source/hu_stuff.c (hunk 1 above)
Comment (rfomin) [reply Source/hu_stuff.c:829]: No, in this case there are more than 1 players, so "+killcount" is added to the string.

Comment (fabiangreffrath) [reply Source/hu_stuff.c:829]: Ah, okay, I have overseen this is within a loop.

**Pull request merged**